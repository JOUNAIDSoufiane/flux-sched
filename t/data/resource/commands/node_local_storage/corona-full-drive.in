match allocate @TEST_SRCDIR@/data/resource/jobspecs/node_local_storage/corona-full-drive.yaml
match allocate @TEST_SRCDIR@/data/resource/jobspecs/node_local_storage/corona-full-drive.yaml
quit
