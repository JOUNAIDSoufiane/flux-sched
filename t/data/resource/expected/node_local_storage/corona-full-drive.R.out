      ------------core0[1:x]
      ------------core1[1:x]
      ------------core2[1:x]
      ------------core3[1:x]
      ------------core4[1:x]
      ------------core5[1:x]
      ------------storage0[1562:x]
      ------------core6[1:x]
      ------------core7[1:x]
      ------------core8[1:x]
      ------------core9[1:x]
      ------------core10[1:x]
      ------------core11[1:x]
      ------------core12[1:x]
      ------------core13[1:x]
      ------------core14[1:x]
      ------------core15[1:x]
      ------------core16[1:x]
      ------------core17[1:x]
      ------------core18[1:x]
      ------------core19[1:x]
      ------------core20[1:x]
      ------------core21[1:x]
      ------------core22[1:x]
      ------------core23[1:x]
      ---------socket0[1:x]
      ------corona116[1:s]
      ---cluster0[1:s]
INFO: =============================
INFO: JOBID=1
INFO: RESOURCES=ALLOCATED
INFO: SCHEDULED AT=Now
INFO: =============================
INFO: =============================
INFO: No matching resources found
INFO: JOBID=2
INFO: =============================
